tasks/services/works.js

```diff
@@ -18,15 +18,58 @@ const hooks = {
   before: {},
   after: {
     create: [
-      iff(completeOrderSetupTaskWork, createCastOrderIntentTaskPlan)
+      iff(taskRecipeIsCompleteOrderSetup, createCastOrderIntentTaskPlan),
+      iff(taskRecipeIsCompleteOrderSetup, createCloseOrderTaskPlan),
+      iff(taskRecipeIsCloseOrder, createCastOrderIntentTaskWorks),
+      iff(taskRecipeIsCloseOrder, createOrderPlans),
+      iff(taskRecipeIsCloseOrder, createViewOrderSummaryTaskPlan)
     ]
   },
   error: {}
 }
 
-function completeOrderSetupTaskWork (hook) {
+function taskRecipeIsCompleteOrderSetup (hook) {
   return hook.data.taskRecipeId === 'completeOrderSetupWithPrereqs'
-  || hook.data.taskRecipeId === 'completeOrderSetup'
+    || hook.data.taskRecipeId === 'completeOrderSetup'
+}
+
+function taskRecipeIsCloseOrder (hook) {
+  return hook.data.taskRecipeId === 'closeOrder'
+}
+
+function createOrderPlans (hook) {
+  // TODO: IK: this is half done, aiming towards creating an orderPlan for each user who created an orderIntent for an order when that order closes
```

Comment:
if this is half done, what's not yet done?

Reply:
old comment, no longer applicable